src/pseudo/load_psp.jl

```diff
@@ -14,7 +14,7 @@ function load_psp(key::AbstractString)
     if startswith(key, "hgh/") || endswith(key, ".hgh")
         parser = parse_hgh_file
         extension = ".hgh"
-    elseif endswith(key, ".upf")
+    elseif startswith(key, "upf/") || endswith(lowercase(key), ".upf")
```

Comment:
Don't put in the `lowercase`. This will cause issues on Windows machines.

Reply:
Or at least do it consistently for all pseudo types.

Reply:
Is it alright to check for ".upf" and ".UPF" separately then?

Reply:
Oh no, that's not what I meant. I meant that either we force people to use one case or we allow both cases for both `upf` and `hgh`. Also for `upf` I'm not sure it makes sense to check the `key` to start with `upf` as we don't ship a `upf` library in the `data` subfolder, right?